frontendUiTable/build.gradle

```diff
@@ -3,6 +3,8 @@ dependencies {
   vavr()
 
   implementation project(':backendApi')
+  implementation(project(':backendRoot'))
```

Comment:
Whoops... can we avoid that dependency? :/

Reply:
PR #213 was specifically dedicated to removing it :)

Reply:
Hmm, strange... I didn't add it (at least not intentionally). Anyway, removed.